.gitignore

```diff
@@ -5,3 +5,10 @@ uvicorn.egg-info/
 venv/
 htmlcov/
 site/
+
+# mypy
+.mypy_cache/
+
+# editors
+.vscode/
+.idea/
```

Comment:
Editor ignores should stay in folks user specific gitignores.
The mypy one can go below .coverage without needing its own comment.

Reply:
Oh didn't know about that, nice to learn something new. I'll update the PR